src/test/java/scm/address/logic/commands/FindAndExportCommandTest.java

```diff
@@ -24,15 +24,14 @@ public class FindAndExportCommandTest {
     @Test
     public void execute_validTagAndFilename_success() throws Exception {
         Path filePath = testFolder.resolve("output.json");
-        FindAndExportCommand command = new FindAndExportCommand("friends", null, null, filePath.toString());
+        FindAndExportCommand command = new FindAndExportCommand("friends", null, null, filePath.toString(), "json");
```

Comment:
It would be good if an enum is used in place of `json` I think?

Reply:
I made a `FileFormat` class to hold constants (of file formats) instead of making ENUMS. The `FileFormat` class also w hosts the `getFileFormat` function which reads a file and and returns the file format. They serve similar purposes.